dockerfiles/theia/Dockerfile

```diff
@@ -27,6 +27,9 @@ ARG THEIA_COMMIT_SHA=''
 
 ENV NODE_OPTIONS="--max-old-space-size=4096"
 
+# avoid any linter/formater/unit test
+ENV SKIP_LINT=true SKIP_FORMAT=true SKIP_TEST=true
```

Comment:
I believe the default should be `false`. Why would we build docker images without checking, but not allow local build without it?

Reply:
because it's checked on another job (and it allows to make the builds as fast as possible)